src/content/developers/docs/layer-2-scaling/index.md

```diff
@@ -4,6 +4,7 @@ description: An introduction to the different scaling options currently being de
 lang: en
 sidebar: true
 incomplete: true
+sidebarDepth: 3
 ---
 
 Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off chain. Transaction speed suffers when the network is busy which can make the user experience poor for certain types of dapps. And as the network gets busier, gas prices increase as transaction senders aim to outbid each other. This can make using Ethereum very expensive.
```

Comment:
```suggestion
Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off the main Ethereum chain. Transaction speed suffers when the network is busy which can make the user experience poor for certain types of dapps. And as the network gets busier, gas prices increase as transaction senders aim to outbid each other. This can make using Ethereum very expensive.
```